app/models/movie.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Movie < ApplicationRecord
+  validates :title, presence: true
```

Comment:
You can pass to [`validates`](https://api.rubyonrails.org/v5.1.7/classes/ActiveModel/Validations/ClassMethods.html#method-i-validates) more than one attribute. So you can write it in that way: `validates :title, :description, presence: true` etc.

Feel free to ignore this comment, it's just a hint. The final decision is up to you and your personal style.

Reply:
Thank you, I'll keep it that way for readability's sake. I know there's not that much attributes here to validate, but I'm just exercising good habits here. ;)